crd.Dockerfile

```diff
@@ -0,0 +1,12 @@
+FROM alpine as builder
+ARG KUBE_VERSION=v1.21.2
+ARG ARCH=amd64
```

Comment:
where is this arg set? see Dockerfile for multi arch builds https://github.com/open-policy-agent/gatekeeper/blob/master/Dockerfile

Reply:
I have updated to use builtkit ARGs instead. This is used to pull down the right kubectl for each arch.